benchmark/util/promisify.js

```diff
@@ -0,0 +1,77 @@
+'use strict';
+
+const common = require('../common');
+const util = require('util');
+const fs = require('fs');
+const zlib = require('zlib');
+const dns = require('dns');
+
+const bench = common.createBenchmark(main, {
+  n: [4e6],
+  fn: [ 'fsStat', 'zlibGzip', 'dnsLookup', 'timersSetTimeout' ]
+});
+
+function main({ n, fn }) {
+  switch (fn) {
+    case 'fsStat':
+      fsStat(n).catch(console.log);
+      break;
+    case 'zlibGzip':
+      zlibGzip(n).catch(console.log);
+      break;
+    case 'dnsLookup':
+      dnsLookup(n).catch(console.log);
+      break;
+    case 'timersSetTimeout':
+      timersSetTimeout(n).catch(console.log);
```

Comment:
Hm, why are these logged? In case a benchmark is run with compare this is going to result in faulty csv output. If something goes wrong, the rejection is better.

Reply:
I logged them because another benchmark working with promises does that for rejections: https://github.com/nodejs/node/blob/master/benchmark/fs/bench-stat-promise.js#L27

If still wanted, I can remove this `.catch()`

Reply:
@AndreasMadsen is there a way to return such errors to the benchmark runner? Because just logging will not be detected depending on how the benchmark is executed.